training/components.py

```diff
@@ -6,8 +6,9 @@
 from utils.consts import PYTHON_IMAGE
 from typing import Optional
 
-@dsl.component(base_image=PYTHON_IMAGE,
-               packages_to_install=["instructlab-training@git+https://github.com/instructlab/training.git"])
```

Comment:
I think we need to keep the @dsl.component for the data_processing_op. Otherwise I'm getting the following error trying to compile the pipeline. 

`TypeError: data_processing_op() missing 1 required positional argument: 'processed_data'`

Reply:
Sorry, fixed now.